tools/vuln_valid/index.js

```diff
@@ -24,7 +24,14 @@ const npmModel = joi.object().keys({
   updated_at: joi.string().regex(/^\d{4}-\d{2}-\d{2}$/).required().isoDate(),
   title: joi.string().required(),
   title: joi.string().max(150).regex(/^[^\n]+$/).required(),
-  author: joi.string().allow(null).required(),
+  author: joi.object().required(),
+  author: joi.object().keys(
+    {
+      name: joi.string().required(),
+      username: joi.string().optional().allow(null),
```

Comment:
Should username and website be required, but allowed to be null?
I kind of think this makes more sense.

Reply:
It helps consistency through the reports format so that's ok with me.